cognite/client/_api/transformations/__init__.py

```diff
@@ -502,7 +502,7 @@ def preview(
         self,
         query: str | None = None,
         convert_to_string: bool = False,
-        limit: int = 100,
+        limit: int | None = 100,
```

Comment:
- Does it work to pass None for all?

- Also, I see that the API spec uses 10k for inferSchemaLimit instead of the 1k in the SDK. Perhaps update as well? 

- Looks like `timeout` i also missing from the params

Reply:
Yes it work for `None`, will fix the other issues.

Reply:
Cool, I wonder why we call it preview if you can page through unlimited results? 🤔

Reply:
I think because you are actually not writing the results to the destination.